src/main/java/com/gmail/shablinski93/repository/impl/DishRepoImpl.java

```diff
@@ -101,11 +90,13 @@ public List<UUID> getIngredientsIdForDish(Connection connection, String dishId)
         try (PreparedStatement preparedStatement = connection.prepareStatement
                 (SELECT_FROM_INGREDIENTS_ID_FOR_DISH)) {
             preparedStatement.setString(1, dishId);
-            ResultSet resultSet = preparedStatement.executeQuery();
-            List<UUID> list = new ArrayList<>();
-            while (resultSet.next()) {
-                String item = resultSet.getString("main_ingredient_id");
-                list.add(UUID.fromString(item));
+            List<UUID> list;
+            try (ResultSet resultSet = preparedStatement.executeQuery()) {
```

Comment:
It's too complicated when you put try inside try, deeper and deeper. That's an anti-pattern. If you cannot refactor it in a plane code it's better to extract it as a private method for logic simplification.